### ISSUE TITLE
Web-Extension Polyfill required?

### ISSUE THREAD

**Issue by ahmadayubi** [Maps4HTML/mapml-extension#2]
Currently the Mozilla's [webextension-polyfill](https://github.com/mozilla/webextension-polyfill) is not used but if it's ever reintroduced re-add `@babel/core` package to  `package.json`.

This polyfill may also not behave as expected with Manifest V3.

**Comment by Malvoz** [comment]
There's an issue tracking Manifest v3 support: https://github.com/mozilla/webextension-polyfill/issues/329

**Comment by bershanskiy** [comment]
Firefox does not support Manifest V3 manifest.json file format yet, so simple polyfill would not be sufficient. As is, extension code is fairly small so it should be easy to just introduce browser-specific code and make a simple build script. I started a related discussion ["Are non-Chrome versions planned?"](https://github.com/Maps4HTML/mapml-extension/discussions/33).

**Comment by prushforth** [comment]
Yes. If I recall correctly, we discussed building for Firefox but decided to wait until move was supported there. We were not familiar enough with extension APIs to know what the issues might be in maintaining two versions for different browsers. I am willing to follow your lead in this.
